pkg/antctl/raw/multicluster/create/clusterclaim.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type clusterClaimOptions struct {
+	namespace    string
+	clusterSetID string
+	clusterID    string
+}
+
+var clusterClaimOpt *clusterClaimOptions
+
+var clusterClaimExamples = strings.Trim(`
+# Create two ClusterClaims, one for the leader or member cluster and another for the ClusterSet
+  $ antctl mc create clusterclaim --cluster-id <CLUSTER_ID> --clusterset-id <CLUSTERSET_ID> -n <NAMESPACE>
```

Comment:
I saw other commands use "--clusterset" and "CLUSTERSET". We should converge all commands to one style.

No strong opinion, but I slightly prefer "--clusterset" and "CLUSTERSET_ID". Let me add a comment to all occurrences with this style.